CONTRIBUTING.md

```diff
@@ -142,3 +142,77 @@ sense for any commit, e.g.
     Acked-by:
     Sponsored-by:
 ```
+
+## Pull requests
+
+Pull requests with patches, improvements, new features are a great help.
+Please keep them clean from unwanted commits.
+
+Follow the steps to get your work included in the project.
+
+1. [Fork](https://help.github.com/fork-a-repo/) the project, clone your fork,
+   and add the fossology remote:
+
+   ```bash
+   # Clone your fork of the repo into the current directory
+   git clone https://github.com/<your-username>/fossology.git
+   # Navigate to the cloned directory
+   cd fossology
+   # Assign the original repo to a remote called "upstream"
+   git remote add upstream https://github.com/fossology/fossology.git
+   ```
+
+2. Get the latest changes from upstream:
+
+   ```bash
+   git checkout master
+   git pull upstream master
+   ```
+
+3. Create a new branch from the main master branch to contain your changes.
+   Best way is to call is to follow the type described in **Git Commit Conventions**
+   stated above: `<type>/<scope>-<topic>`
```

Comment:
If you are creating a PR from a fork, the username will be displayed in from of the branch name. For e.g.: you can check here #1130
Otherwise, you can easily see the person creating the PR. Is it really required?

Reply:
>  This is usefull if one works in the CLI and does not wan't to checkout the branch to find out who is working on it.

You do not need to checkout the branch, you can directly call `git log <branch>` to check the last commits.

Reply:
Done, please check.